content/analytics/data-views/data-view-link-fields.md

```diff
@@ -8,6 +8,8 @@ Data items included in a data view may have slight differences in property namin
 
 Data views can overcome property naming differences by linking these similar properties into a single data field. This applies to stream properties referenced by Id or by name, and to stream metadata keys. Asset properties can also be linked.
 
+When data fields are linked, AVEVA Data Hub converts the data types in the linked into compatible data types. For more information, see <xref:data-view-data-type-conversion>.
```

Comment:
```suggestion
When data fields are linked, AVEVA Data Hub converts the data types in the linked fields into compatible data types. For more information, see <xref:data-view-data-type-conversion>.
```

Reply:
I ended up taking this part out on SME recommendation.